src/main/java/de/unistuttgart/t2/cart/repository/TimeoutCollector.java

```diff
@@ -20,12 +21,17 @@
  * @author maumau
  */
 @Component
-public class TimeoutCollector {
+public final class TimeoutCollector {
+
+    private static final Logger LOG = LoggerFactory.getLogger(TimeoutCollector.class);
 
     private final long TTL; // seconds
     private final int taskRate; // milliseconds
-
-    private final Logger LOG = LoggerFactory.getLogger(getClass());
+    final Runnable cleanup = () -> {
```

Comment:
maybe a method would be a cleaner solution

Reply:
Yep, I also noticed that one later in `inventory`(?) where the exact same thing needs to be present.